Gemfile

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+gemspec
```

Comment:
- gemspec
+ gem "github-pages", group: :jekyll_plugins

Will promote the remote-theme approach rather than the gemspec approach

Reply:
Using the "github-pages" gem instead of plain Jekyll can be done in the gemspec, but I didn't think it needed anything from that gem besides the right Jekyll version. I just copied it from the pages-gem repo :).
I believe they aren't contradictory, but I only went over the docs late last night